df_ref.csv

```diff
@@ -1,9 +1,9 @@
-,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,sub-test002_T2w_seg-manual.nii.gz,,,,,,,,,
-,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,anat,.nii.gz,test003,T2w,sub-test003_T2w.nii.gz,,,,,,,,,-
-,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,sub-test003_T2w_seg-manual.nii.gz,,,,,,,,,
-260,1,12:33:7.032500,320,SPINE,N4_VE11C_LATEST_20160120,dcm2niix,v1.0.20180622 (JP2:OpenJPEG) GCC6.1.0,167006,,0.00372,,9,"[0, 1, 0, -0.00523601, 0, -0.999986]","['ORIGINAL', 'PRIMARY', 'M', 'NORM', 'DIS3D', 'DIS2D']",ROW,Chemin_Queen-Mary_4565_Montreal_District_CA_H3W_1W5,IUGM,,1,3D,3,Siemens,Prisma_fit,MR,2,1,HFS,81.25,259,,1,150,spine_generic,T1w,%SiemensSeq%_tfl,"HC1-7;NC1,2",HeadNeck_64,320,2,0.0601922,IR,GR_IR,_tfl3d1_16,SK_SP_MP,T1w,2,"[620, -7194, -9127, 77, 28, -20, -10, -23]",1,syngo_MR_E11,MRC35049,266.247,anat,.nii.gz,unf01,T1w,sub-unf01_T1w.nii.gz,-,unknown,-,unknwon,unknown,unknown,unknown,unknwon,unknown
+260,1,12:33:7.032500,320,SPINE,N4_VE11C_LATEST_20160120,dcm2niix,v1.0.20180622 (JP2:OpenJPEG) GCC6.1.0,167006,,0.00372,,9,"[0, 1, 0, -0.00523601, 0, -0.999986]","['ORIGINAL', 'PRIMARY', 'M', 'NORM', 'DIS3D', 'DIS2D']",ROW,Chemin_Queen-Mary_4565_Montreal_District_CA_H3W_1W5,IUGM,,1,3D,3,Siemens,Prisma_fit,MR,2,1,HFS,81.25,259,,1,150,spine_generic,T1w,%SiemensSeq%_tfl,"HC1-7;NC1,2",HeadNeck_64,320,2,0.0601922,IR,GR_IR,_tfl3d1_16,SK_SP_MP,T1w,2,"[620, -7194, -9127, 77, 28, -20, -10, -23]",1,syngo_MR_E11,MRC35049,266.247,anat,.nii.gz,unf01,T1w,sub-unf01_T1w.nii.gz,-,unknown,-,unknown,unknown,unknown,unknown,unknown,unknown
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,sub-unf01_T1w_seg-manual.nii.gz,,,,,,,,,
-320,1,12:39:2.805000,320,SPINE,N4_VE11C_LATEST_20160120,dcm2niix,v1.0.20180622 (JP2:OpenJPEG) GCC6.1.0,167006,2.5e-06,0.12,76,120,"[0, 1, 4.897e-12, 0, 4.897e-12, -1]","['ORIGINAL', 'PRIMARY', 'M', 'NORM', 'DIS3D', 'DIS2D']",COL,Chemin_Queen-Mary_4565_Montreal_District_CA_H3W_1W5,IUGM,Department,,3D,3,Siemens,Prisma_fit,MR,3,1,HFS,100,422,0.8,1,625,spine_generic,T2w,%SiemensSeq%_tse_vfl,"HC5-7;NC1,2;SP1",Spine_32,320,1.5,0.533389,PFP,SE,_spcR_100,SK_SP_OSP,T2w,3,"[620, -7194, -9127, 77, 28, -20, -10, -23]",0.8,syngo_MR_E11,MRC35049,283.131,anat,.nii.gz,unf01,T2w,sub-unf01_T2w.nii.gz,-,unknown,-,unknwon,unknown,unknown,unknown,unknwon,unknown
+320,1,12:39:2.805000,320,SPINE,N4_VE11C_LATEST_20160120,dcm2niix,v1.0.20180622 (JP2:OpenJPEG) GCC6.1.0,167006,2.5e-06,0.12,76,120,"[0, 1, 4.897e-12, 0, 4.897e-12, -1]","['ORIGINAL', 'PRIMARY', 'M', 'NORM', 'DIS3D', 'DIS2D']",COL,Chemin_Queen-Mary_4565_Montreal_District_CA_H3W_1W5,IUGM,Department,,3D,3,Siemens,Prisma_fit,MR,3,1,HFS,100,422,0.8,1,625,spine_generic,T2w,%SiemensSeq%_tse_vfl,"HC5-7;NC1,2;SP1",Spine_32,320,1.5,0.533389,PFP,SE,_spcR_100,SK_SP_OSP,T2w,3,"[620, -7194, -9127, 77, 28, -20, -10, -23]",0.8,syngo_MR_E11,MRC35049,283.131,anat,.nii.gz,unf01,T2w,sub-unf01_T2w.nii.gz,-,unknown,-,unknown,unknown,unknown,unknown,unknown,unknown
```

Comment:
thanks for the fix!

Reply:
what's this df_ref file about by the way?

Reply:
This file is used in the tests for the new loader (test_loader.py) to compare the import if the bids dataframe.